docs/hyperion/user-guide/deploy-charts/deployment-of-charts.md

```diff
@@ -0,0 +1,75 @@
+# Chart Operations 
+
+## Introduction
+
+Discover, Create, Deploy, Update, Upgrade, Delete charts.
+
+## 1. Discover the chart from the Chart Store
+
+Select the `Charts` section from the left pane, you will be landed to the `Chart Store` page. 
+Search `nginx` or any other charts in search filter.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/hyperion/user-guide/deploy-charts/overview-of-charts/deployment-of-charts/charts-1.jpg)
+
+Click on chart and it will redirect you to `Chart Details` page where you can see number of instances deployed by using the same chart.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/deploy-chart/deployment-of-charts/charts-2.jpg)
+
+## 2. Configure the Chart
+
+After selecting the version and values, click on `Deploy`
+
+Enter the following details, to deploy chart:
+
+| Key | Description |
+| :--- | :--- |
+| `App Name` | Name of the Chart Unique|
+| `Project` |  Project in which you want to deploy the chart |
```

Comment:
Is Project required in hyperion mode too?

Reply:
No. removed the project field in doc.